sklearn/metrics/ranking.py

```diff
@@ -197,18 +141,20 @@ def average_precision_score(y_true, y_score, average="macro",
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
-    interpolation : string ['linear' (default), 'step']
-        Determines the kind of interpolation used when computed AUC. If there
-        are many repeated scores, 'step' is recommended to avoid under- or
-        over-estimating the AUC. See `Roam Analytics blog post
-        <https://github.com/roaminsight/roamresearch/blob/master/BlogPosts/
-        Average_precision/Average_precision_post.ipynb>`
-        for details.
-
-        ``'linear'``:
-            Linearly interpolates between operating points.
-        ``'step'``:
-            Uses a step function to interpolate between operating points.
+    interpolation : None (default), or string ['eleven_point']
```

Comment:
I was just copying the style from other places that includes acceptable strings in a list. But they had multiple options so I guess that made more sense. Should it read `interpolation : None (default), or string 'eleven_point'` or `interpolation : None (default), or string ['eleven_point', ]`?